modules/ROOT/pages/microsoft/microsoft-365-ops-connector.adoc

```diff
@@ -82,49 +92,64 @@ You can use this connector in Anypoint Studio by first downloading it from Excha
+|Consumer Secret |The app secret that you created in the app registration portal for your app. This should not be used in a native app, because client secrets cannot be reliably stored on devices. This is required for web apps and web APIs, which have the ability to store the client secret securely on the server side. You can find this in the Azure Portal (portal.azure.com).
+|Authorization URL |Tenant specific OAuth2.0 endpoint; format: `+https://login.microsoftonline.com/{tenant}/oauth2/authorize+` (follow instructions presented in the previous link for specific information on how to obtain the endpoint)
+|Access token URL |Tenant specific endpoint for requesting the access token; format: `+https://login.microsoftonline.com/{tenant}/oauth2/token+` (follow instructions presented in the previous link for specific information on how to obtain the endpoint).
+|Listener config |Name of the HTTP listener instance created in studio flow
+|Callback path |The URL of your app, where authentication responses can be sent and received by your app (follow instructions presented in the above link for specific information on how to obtain the endpoint).
+|Authorize path |The URL which triggers the OAuth mechanism.
+|External callback URL |Optional: Configure if the callback URL is not accessible directly (example: when using proxies) example format: `+http://localhost:8081/callback+`
+|===
+
+image::microsoft/microsoft-365-ops-as-oauth2.png[]
+
+=== Configure OAuth 2.0 Username Password
+
+[%header,cols="30s,70a"]
+|===
+|Field |Description
+|Username |Username used to initialize the session.
+|Password |Password used to authenticate the user.
+|Resource |The app ID URI of the web API (secured resource). This must be the root URI, without specifying the version. Example: `+https://YOUR_ORG.operations.dynamics.com/+`
+|Client ID |The app ID assigned to your app when you registered it with Azure Active Directory. You can find this in the Azure Portal (portal.azure.com). Click Active Directory, click the directory, choose the app, and click Configure.
+|Client Secret |The app secret that you created in the app registration portal for your app. This should not be used in a native app, because client secrets cannot be reliably stored on devices. This is required for web apps and web APIs, which have the ability to store the client secret securely on the server side. You can find this in the Azure Portal (portal.azure.com).
+|Token Request Endpoint |The token endpoint that is called to get the access token. Example: `+https://login.windows.net/TENANT_ID/oauth2/token+` where TENANT_ID is the Azure Active Directory ID.
+|===
 
-== Use Case: Studio
+image::microsoft/microsoft-365-ops-as-oauth-username-pass.png[]
 
-. Listener(HTTP) - accepts data from HTTP requests.
+
+== Use Case: Accept Data and Output Into JSON
```

Comment:
```suggestion
== Use Case: Accept Data and Output into JSON
```

Reply:
This title should be followed by text that sets the scenario for this use case and explains to your audience how examining it will help them achieve their own particular goals.

As it is now written, I as a reader do not understand how I got from the title to the numbered list (why is it numbered? Is this a task or a machine process? Explain) and illustration. 

Suggest introducing the illustration, showing it, and then introducing the numbered steps shown by it.
Then consider introducing the XML syntax example that follows it, without giving it a separate heading.